x-pack/plugin/slm/src/test/java/org/elasticsearch/xpack/slm/action/ReservedSnapshotLifecycleStateServiceTests.java

```diff
@@ -111,9 +111,10 @@ public void testValidationFails() {
               }
             }""";
 
+        expectThrows(IllegalArgumentException.class, () -> processJSON(action, prevState, badPolicyJSON)).getMessage();
```

Comment:
I'm not following the importance of this `expectThrows` versus the one after this inside the `assertThat` which looks identical to me.

Reply:
Whoops! Yeah, that was me messing to understand what was happening with the following test case and shouldn't still be there!